src/database/db.go

```diff
@@ -2,59 +2,49 @@ package database
 
 import (
 	"context"
-	"fmt"
 	"log"
-	"os"
 	"time"
 
-	"github.com/joho/godotenv"
+	"github.com/fullstacktf/personal-nutritionist-backend/env"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
 const connectTimeout = 5
 
 func getConnectionURI() string {
-	err := godotenv.Load("../.env")
-	if err != nil {
-		log.Fatal("Error loading .env file 💣")
-	}
-
-	username := os.Getenv("MONGO_USERNAME")
-	password := os.Getenv("MONGO_PASSWORD")
-	database := os.Getenv("MONGO_DATABASE")
-
-	return "mongodb://" + username + ":" + password + "@localhost/27017/" + database + "?authSource=admin"
+	env.LoadEnv()
```

Comment:
Maybe we will have trouble with CD Branch here we must try docker image and containers works correctly